apps/mull-ui/src/app/app.tsx

```diff
@@ -118,6 +119,11 @@ export const App = () => {
               <Redirect to={ROUTES.ANNOUNCEMENTS.url} />
             </PrivateRoute>
           </PrivateRoute>
+          <PrivateRoute path={ROUTES.PROFILE.profile}>
+            <div className="page-container with-sub-nav-bar">
```

Comment:
The `with-sub-nav-bar` class adds spacing to the container to account for a subnavbar but your page doesn't have one so it shouldn't have this class
```suggestion
            <div className="page-container">
```

Reply:
Done.